server/config/routes.js

```diff
@@ -54,7 +54,7 @@ module.exports = function(app, passport) {
   // fetched and seed our stores with data.
   // App is a function that requires store data and url to initialize and return the React-rendered html string
   app.get('*', function (req, res, next) {
-    App(req, res);
+    App.default(req, res);
```

Comment:
@kaizvn We could either roll with the existing solution you have, or try [this](http://stackoverflow.com/a/33683495/1710968). I'm happy with either, as long as we document it saying this is `expected behaviour is ES6` with a link